webpack/css.js

```diff
@@ -0,0 +1,12 @@
+module.exports = function() {
```

Comment:
You won't write css in css files, you'll use styles components. So remove all plugins connected with css